nisystemlink/clients/dataframe/_data_frame_client.py

```diff
@@ -33,6 +33,41 @@ def api_info(self) -> models.ApiInfo:
         """Returns information about available API operations."""
         ...
 
+    @get(
+        _BASE_PATH + "/tables",
+        args=(
+            Query("take"),
+            Query("id"),
+            Query("orderBy"),
+            Query("orderByDescending"),
+            Query("continuationToken"),
+            Query("workspace"),
+        ),
+    )
+    def list_tables(
+        self,
+        take: Optional[int] = None,
+        id: Optional[List[str]] = None,
```

Comment:
Looks like it's serializing it the first way `?id=1&id=2` (and the service handles it correctly). Do we need to fix it to be comma separated?

Reply:
@spanglerco - This is the last remaining open issues on this PR and we're wanting to get this submitted to test out the semantic release stuff. I spoke with Carson offline and we agreed that since this works in its existing state we can go ahead and submit this change with this behavior, especially since we're still publishing prerelease packages. I do think we should consider making this change though.

@mure - Please create a follow-up task under your user story to discuss whether or not we should address this in a separate PR.

Reply:
If there's an easy way to get uplink to send it as a comma-separated list, I would prefer that. But if it's a lot of work, we probably don't need to worry about it. We may want to add tests to the DFS to ensure we don't break this way of doing it. We're getting it for free from ASP.NET, so if we were to switch frameworks in the future, it may break without us thinking about it.